src/main/java/com/project/bookstore/controller/BookController.java

```diff
@@ -27,4 +28,15 @@ public BookDto findById(@PathVariable Long id) {
     public BookDto save(@RequestBody CreateBookRequestDto requestDto) {
         return bookService.save(requestDto);
     }
+
+    @PutMapping("/{id}")
+    public void update(@PathVariable Long id, @RequestBody CreateBookRequestDto requestDto) {
+        bookService.updateById(id, requestDto);
+    }
+
+    @ResponseStatus(HttpStatus.NO_CONTENT)

```

Comment:
1: In future you will have problem with testing void methods. 
Consider returning result. 

2: Use ResponseEntity<T> over @ResponseStatus. It's modern way to manipulate returning HTTP message. 
```suggestion
    @ResponseStatus(HttpStatus.NO_CONTENT)

```